tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+- name: Check that either interface or connection name is provided
```

Comment:
I'm just gonna go ahead and assume that all of this is reasonable and sane.

Reply:
I had another scenario that attempted to assert against it, but it proved very difficult to successfully run a role that fails on purpose. I plan to tinker with that a little more, but wanted to get the code up first to make it a little easier to file an issue about it.